docker/mongo/setup.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+tar -xf /to_restore/wtc_console_dev.tar.gz
```

Comment:
We should probably avoid adding binaries to the repo. I'm not sure how to handle it for `mongodb` though, for a SQL database a SQL CREATE/INSERT script with a small sample is enough and still plaintext, maybe for `mongodb` there is a similar solution. Do you know what is the smallest number of entries for the collections that is still useful?

Reply:
That's a good question. All I really want right now is to duplicate the thing where steps are showing up in multiple PrepIDs. That's happening here.

If we want to literally just keep this out of the git repo, but are okay with all the data, I can put this on Docker Hub instead and point the docker-compose.yml there.

Reply:
Yup. Or maybe doing a `wget` to the `tar.gz` and place it somewhere else until we find a better solution. What do you think?